_docs/technical/2017-01-07-wallet.md

```diff
@@ -68,51 +68,8 @@ complete APIs.
 If the event requests fail, there is a `WalletError` type, which
 is simply a wrapper over `Text` to show what happened.
 
-Please note that:
+You can obtain wallet's API documentation from the `cardano-node` itself. This is a Swagger-based documentation,
```

Comment:
How about we create a separate page where we can copy the generated documentation? That would solve all the issues.